stable/elasticsearch-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: elasticsearch-exporter
-version: 1.10.0
+version: 1.10.1
```

Comment:
not sure if changing a config name should be only a patch update... I would expect it to be bumped to at least `1.11.0` or maybe even `2.0.0`...

Reply:
I am unsure what config you are talking about I fixed the README to be what that value is in the deployment 
https://github.com/helm/charts/blob/master/stable/elasticsearch-exporter/templates/deployment.yaml#L77

Reply:
yes, but for someone using the old config (`es.ssl.ca.pemPath`) this release is breaking, as they'll need to change their local values.

A patch bump ususally represents non-breaking changes...

Reply:
If anyone was using es.ssl.ca.pemPath in the values file it would have never worked. It's a bug from the last uprev which was a minor change adding the feature. I am just trying to fix that minor version. I can uprev again, but I didn't think it was necessary.

Reply:
ah ok then.